run.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-# install packages 
-apt-get install -qqy pigpio
+apt-get install -qqy libpigpio1=1.79-1+rpt1 libpigpio-dev=1.79-1+rpt1 
```

Comment:
I'm fine with pinning versions here, but I really think it needs to include the versions of packages you actually use DIRECTLY. The module doesn't even use libpigpio1 or libpigpio-dev itself. It only depends DIRECTLY on libpigpiod-if2-1 (and requires pigpiod to be installed.) If you're going to force versions on only two packages, it should be these two. Everything is superfluous.